target/linux/ramips/mt7621/base-files/etc/board.d/01_leds

```diff
@@ -150,6 +150,10 @@ linksys,ea8100-v2)
 	ucidef_set_led_netdev "lan4" "lan4 link" "green:lan4" "lan4" "link"
 	ucidef_set_led_netdev "wan" "wan link" "green:wan" "wan" "link"
 	;;
+mercury,km08-708h)
+	ucidef_set_led_netdev "lan2" "lan2 link" "green:lan2" "lan2" "link tx rx"
+	ucidef_set_led_netdev "wlan" "wlan link" "green:wlan" "wlan0" "link tx rx"
```

Comment:
This should not be needed with your `linux,default-trigger = "phy0radio";` definition.

Reply:
still needed. it did not generate led config when i removed it.

Reply:
For the LAN port it is needed, for WLAN it should work without a configuration.